src/bridge/askpass.c

```diff
@@ -197,8 +197,9 @@ main (int argc,
             {
               if (g_str_equal (field, cookie) && g_str_equal (command, "authorize"))
                 {
+                  /* If we don't have a password, the response is an empty string; abort askpass with non-zero exit */
```

Comment:
If this is meant to check if it is empty "" or NULL wouldn't we want to use `g_strcmp0(str, "") != 0`?

Reply:
Guaranteed to be a string by `cockpit_json_get_string (reply, "response", "", &response))`, so this is a matter of taste.

Reply:
It can't be `NULL` (as you found out), and the code below also already used it without a NULL check. Then strcmp(s, "") or `*s` as a boolean are equivalent. I don't mind much to change it if you find it easier to read, though!